src/js/constants/message.js

```diff
@@ -3,3 +3,9 @@ export const SEND_STATUS = {
     FAILED: 'failed',
     SENT: 'sent'
 };
+
+export const LOCATION_ERRORS = {
```

Comment:
They're generated by the browser, I figured they were nice to have even if unused.